components/organisms/Dashboard/dashboard.tsx

```diff
@@ -15,6 +15,7 @@ import roundedImage from "lib/utils/roundedImages";
 import { useTopicContributions } from "lib/hooks/useTopicContributions";
 
 type ContributorPrMap = { [contributor: string]: DbRepoPR };
+export type PrStatusFilter = "open" | "closed" | "all";
```

Comment:
this is shadowing the use of enums while still checking for literal values below; can we have the `prStateFilter` just pass the pr values around so we avoid this mechanic?

Alternatively, if this is a global type, it should be set up in the global types files. If the enum itself is required, the inline checks for things like `open` should also be replaced with the enum, however this use is unprecedented due to us using global envs.

Looking at the rest of the PR, this is forced usage at the moment and should be reworked.

Reply:
sorry but I didn't understand the wording of the statement
> can we have the prStateFilter just pass the pr values around

What do you mean by "pass the pr values around"? Like make the type string instead of those 3 values?

Reply:
> Alternatively, if this is a global type, it should be set up in the global types files.

Well yes it is a global type, as any PR `status` prop would be either "open" or "closed", but I didn't want to make a global change.

Reply:
Yes, we have a type, inline string usage and also numeric values. The component does not need to translate numeric into string (that would be why using enums is superior)

Reply:
This change is in PR #736 though. This pr just incorrectly included that PRs commits in it.

Reply:
> > Alternatively, if this is a global type, it should be set up in the global types files.
> 
> Well yes it is a global type, as any PR object `status` prop would be either "open" or "closed", but I didn't want to make a global change.

@brandonroberts what are your thoughts on this? I know enums are questionable usage but in any case, we should simplify to 1 type of usage for these values rather than 3 :D